src/App.tsx

```diff
@@ -21,75 +29,40 @@ export const App: React.FC = () => {
             data-cy="perPageSelector"
             id="perPageSelector"
             className="form-control"
+            defaultValue={selectedValue}
+            onChange={(e) => {
+              setSelectedValue(e.currentTarget.value);
+              setActive(1);
+            }}
```

Comment:
Don't write functions inside JSX. It's better to move it to a separate function

Reply:
think that it is not necessary here. An additional function will only add more confusion